README.md

```diff
@@ -30,6 +30,13 @@ sudo apt-get install libxcb-render0-dev libffi-dev python-dev python-cffi
 pip install flashfocus
 ```
 
+### Fedora
+
+```
+dnf install libffi-devel python-devel python-cffi
+sudo pip install flashfocus
```

Comment:
Could you remove the sudo in this line? prefer not to be opinionated about where people are installing the package.